package.json

```diff
@@ -7,10 +7,14 @@
   },
   "dependencies": {
     "nan": "2.x",
-    "tar": "2.x"
+    "tar": "2.x",
+    "nodereport": "1.x",
+    "heapdump": "0.x"
   },
   "bundleDependencies": [
-    "tar"
+    "tar",
+    "nodereport",
+    "heapdump"
   ],
```

Comment:
These don't need to be bundled, we only bundle tar because it's used by the install script (extract_all_binaries.js) and npm doesn't guarantee that deps will be installed before the install script is run.